test/box-tap/check_merge_source.c

```diff
@@ -37,6 +36,9 @@ luaT_check_merge_source(struct lua_State *L, int idx)
 	return *source_ptr;
 }
 
+struct lua_State *
+fiber_lua_state_internal(struct fiber *fiber);
```

Comment:
Why can't you include fiber.h which contains this declaration already?

Reply:
Because in this case the `enum say_level` will be declared twice (from `module.h` and from `fiber.h -> diag.h -> say.h`).

Reply:
Well, ok, why do you need module.h here then?

Reply:
To use `box_error_last` because I don't want to export `diag_get`.

Reply:
Ok, why can't you make `#include "diag.h"` and use it? It was already included before the patch. Why did you touch this file at all? Can you simply link it with libcore in cmake and leave it as is?

Reply:
I had a conversation with Vlad and it looks like he won the fight. We can include `error.h` instead of `module.h` and not export any extra symbols (I mean `diag_get` or `fiber_lua_state_internal`).